src/main/python/car_trecweb.py

```diff
@@ -20,22 +25,48 @@ def writer(p, fp):
     content += (u'<DOCHDR>\n')
     content += (u'\n')
     content += (u'</DOCHDR>\n')
+    if meta_dict and p.para_id in meta_dict:
+        content += (u'<TITLE>\n')
+        content += (meta_dict[p.para_id]["title"])

```

Comment:
What if a paragraph doesn't have a title, check for this and log it? A warning that the paragraph is missing metadata.   I expect this may happen in a small number of cases; make it clear if it's info, warning, or error.

Reply:
added a warning, was already doing some checking if the metadata wasn't there